packages/generator-superset/generators/plugin-chart/templates/src/types.erb

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Might just want to add this to the file tree in the README... or remove the file tree from the README (not sure how much value it really provides)

Reply:
Good catch. `types.ts` was actually already in the README, but for some reason file was missing before this (probably fell out during the long development/review process). However, `tsconfig.json` was missing, so I added that.